compiler/src/dotty/tools/dotc/core/NameOps.scala

```diff
@@ -280,6 +287,22 @@ object NameOps {
         classTags.fold(nme.EMPTY)(_ ++ _) ++ nme.specializedTypeNames.suffix)
     }
 
+    /** Use for specializing function names ONLY and use it if you are **not**
+     *  creating specialized name from type parameters. The order of names will
+     *  be:
+     *
+     *  `<return type><first type><second type><...>`
+     */
+    def specializedFunction(ret: Type, args: List[Type])(using Context): Name =
+      val sb = new StringBuffer
```

Comment:
We should always use StringBuilder, not StringBufffer. StringBuffer is basically deprecated: https://stackoverflow.com/questions/355089/difference-between-stringbuilder-and-stringbuffer
```suggestion
      val sb = new StringBuilder
```

Reply:
Thanks for the tip 👍